.github/workflows/perf.yaml

```diff
@@ -61,3 +61,10 @@ jobs:
         export WAVE_RUN_E2E_TESTS=1
         export TEST_PARAMS_PATH="tests/kernel/wave/test_param.json"
         pytest -n 1 --capture=tee-sys -vv ./tests/kernel/wave/
+
+    - name: Run e2e tests on MI250
```

Comment:
You can remove this step completely, and just keep the one before it. Just hange its name from Run e2e tests on MI300 to Run e2e tests on AMD GPU or something and remove the `if: "contains(matrix.os, 'mi300') && !cancelled()"` line

Reply:
Can you further explain? If I understand correctly, you want to only have one test? but we need to run both MI250 and MI300 though

Reply:
actually I see what you mean!

Reply:
Done! thanks you again for the tip! :)